modules/misc/news.nix

```
@@ -1679,6 +1679,14 @@ in {
           https://github.com/rafaelmardojai/blanket for more.
         '';
       }
+
+      {
+        time = "2024-06-23T14:34:28+00:00";
+        condition = config.programs.yazi.enable;
```

Only show this when shell integration is enabled.

```suggestion
        condition = with config.programs.yazi; enable && (enableBashIntegration || enableZshIntegration || enableFishIntegration || enableNushellIntegration);
```

---

Nice catch!